1-exercises/E-for-loop/exercise2.js

```diff
@@ -11,23 +11,19 @@
 */
 
 const WRITERS = [
-    "Virginia Woolf",
-    "Zadie Smith",
-    "Jane Austen",
-    "Bell Hooks",
-    "Yukiko Motoya"
-]
-
-const AGES = [
-    59,
-    40,
-    41,
-    63,
-    49
+  "Virginia Woolf",
+  "Zadie Smith",
+  "Jane Austen",
+  "Bell Hooks",
+  "Yukiko Motoya",
 ];
 
-// TODO - Write for loop code here
+const AGES = [59, 40, 41, 63, 49];
 
+// TODO - Write for loop code here
+for (let i = 0; i < WRITERS.length; i++) {
+  console.log(`${WRITERS[i]} is ${AGES[i]} years old`);
```

Comment:
I think it could be even cleaner if you assigned a var names to WRITERS[I] and AGES[I]. Than use those names in the template literals